users/urls.py

```diff
@@ -24,4 +25,7 @@
         PasswordResetAPIView.as_view(),
         name="reset-password",
     ),
+    path(
+        "users/follow/<int:pk>/", UserFollowView.as_view(), name="user-follow"
```

Comment:
use a non-sequential id for lookup

Reply:
The uuid will be set up by @foscraft. It will still be the primary key and the functionality will remain the same. i.e all variable naked id/pk will still be named as such

Reply:
i doubt cause he implemented another field called lookup_id

Reply:
He is implementing it and I doubt it will affect the structure of my code as the falls under the models